### PR TITLE
feat(scopes): Add npm scope support.

### LINKED ISSUE #59: support for scoped packages
The `gulp-load-plugins` module does not support scoped gulp plugins. [npm-scope](https://docs.npmjs.com/misc/scope).

I believe line [20](https://github.com/jackfranklin/gulp-load-plugins/blob/master/index.js#L20) could be changed from `['gulp-*', 'gulp.*']` to `['gulp-*', 'gulp.*', '@*/gulp{.,-}*']`.

### PR DESCRIPTION
**Pull request by hbetts**
Closes #59

### CODE REVIEW

**Comment by hbetts** [comment]
@jackfranklin some of this change (mostly the refactoring) is actually based on your scope branch that I just found.

ping @matijakovacevic

ping @courtnek for code review.

**Comment by jackfranklin** [comment]
@hbetts nice ! I was planning on finding time to get that branch fixed and merged in but you've beaten me to it, thank you.

File: index.js [hunk 1]
```diff
@@ -62,15 +74,27 @@ module.exports = function(options) {
...
-      });
+    return requireName;
+  }
+
+  var scopeTest = new RegExp('^@');
+  var scopeDecomposition = new RegExp('^@(.+)/(.+)');
+
+  multimatch(names, pattern).forEach(function(name) {
+    var hasScope = scopeTest.test(name);
+    var decomposition = scopeDecomposition.exec(name);
```
Comment (courtnek) [inline index.js]: This is not really needed unless we are in a hasScope case. I'd vote to move this down and inline the "requireName" value, but that's totally a style choice. 

I doubt there would be any measurable performance concern.

**Comment by courtnek** [comment]
@hbetts :+1: LGTM. Feel free to submit as is. Left a nit-picky style comment.

**Comment by hbetts** [comment]
I refactored my commit based on @courtnek's suggestion, and I added a change to the `README.md` file to inform consumers of the scope support.

**Comment by jackfranklin** [comment]
Add yourself to the contributors part of the package.json too!

**Comment by hbetts** [comment]
@jackfranklin, I've added another commit to update the contributors list.

**Comment by callumacrae** [comment]
:clap:

**Pull request merged**